plugins/feeds/public/feodo_tracker.py

```diff
@@ -89,35 +89,36 @@ def analyze(self, item):
 
                 soup = BeautifulSoup(s, 'html.parser')
                 res = soup.find_all('table')
-                res = res[1].find_all('td')
+                if len(res) > 1:
+                    res = res[1].find_all('td')
+                    if len(res) >= 8:
```

Comment:
I would rather follow the HTML structure. Start with `find_all('tr')` and then `find_all('td')`. That way is less error-prone (e.g. in its current state the code will break if a column is added to the table)